packages/build-utils/src/prerender.ts

```diff
@@ -12,6 +12,7 @@ interface PrerenderOptions {
   initialStatus?: number;
   passQuery?: boolean;
   sourcePath?: string;
+  experimentalStreamingLambda?: string;
```

Comment:
```suggestion
  experimentalStreamingLambdaPath?: string;
```
Maybe we call this the path to stress that's what it is and not a Lambda output itself like the `lambda` field.

Reply:
That is a better name!